src/viewers/KYCViewer.sol

```diff
@@ -37,6 +37,7 @@ contract KYCViewer is Initializable, UUPSUpgradeable, OwnableUpgradeable, IKYCVi
      */
     function initialize() external initializer {
         __Ownable_init();
+        __UUPSUpgradeable_init();
```

Comment:
@fedealconada we need to deploy a new KYC viewer at some point because I messed up the original deploy because this line was missing